### DISCUSSION TITLE
Sysmon Playbook Alerts look weired

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8893] [Unsupported Versions] [answered]
Hello!

My Sysmon Playbook Alerts look weired in Hunt (SO 2.3.170):

![image](https://user-images.githubusercontent.com/81571847/195018391-0a93d42b-b3ca-46c3-b41c-1bf631c641cf.png)

Any ideas what's going on there?

Cheers, Ben

**Comment by defensivedepth** [comment]
@ben-sec Do you have any custom Plays or Elastalert rules? Which Play is this generated from?

**Reply by ben-sec** [reply]
This one:

![image](https://user-images.githubusercontent.com/81571847/195342033-47b11b88-6b05-4be6-8c32-ef02efc4e601.png)

**Reply by defensivedepth** [reply]
No custom plays or elastalert rules?

**Reply by ben-sec** [reply]
Oh sorry, yes I have some custome Elastalerts and one custom Play.

**Reply by defensivedepth** [reply]
Can you sanitize & drop the details / json / etc for one of those alerts over on a gist & post it here?

**Reply by ben-sec** [reply]
There you go: https://gist.github.com/ben-sec/21ad2c39058a5c6d410b50363a1cce75

**Reply by ben-sec** [reply]
BTW: I received a different Playbook alert today and it looked just fine in Hunt. Maybe it's just the rule?

**Reply by defensivedepth** [reply]
@ben-sec Ya, looking at that rule & how it is being converted --> doesn't look right. I would suggest disabling it for now. I have it on my list to look into it and adjust the field mappings for those fields.

**Comment by ben-sec** [comment, marked as answer]
OK, rule has been deactivated.
